user/languages/java.md

```diff
@@ -130,23 +132,26 @@ install: ant deps
 
 ## Testing Against Multiple JDKs
 
-To test against multiple JDKs, use the `jdk:` key in `.travis.yml`. For example, to test against Oracle JDK 7 and 8 and OpenJDK 6:
+To test against multiple JDKs, use the `jdk:` key in `.travis.yml`. For example, to test against Oracle JDKs 8 and 9, as well as OpenJDK 7:
 
 ```yaml
 jdk:
   - oraclejdk8
-  - oraclejdk7
-  - openjdk6
+  - oraclejdk9
+  - openjdk7
 ```
 {: data-file=".travis.yml"}
 
 > Note that testing against multiple Java versions is not supported on OS X. See the [OS X Build Environment](/user/reference/osx/#JDK-and-OS-X) for more details.
 
-Travis CI provides OpenJDK 6, OpenJDK 7, Oracle JDK 7, and Oracle JDK 8. Sun JDK 6 is not provided, because it is EOL as of November 2012. OpenJDK 8 is available on our Trusty images, specify `dist: trusty` to make use of it.
+Travis CI provides OpenJDK 7, Oracle JDK 8, and Oracle JDK 9.
+
+### Updating Oracle JDK to a recent release
 
-JDK 7 is backwards compatible, we think it's time for all projects to start testing against JDK 7 first and JDK 6 if resources permit.
+Your repository may require a newer release of Oracle JDK than the pre-installed version.
+(You can consult [the list](https://launchpad.net/~webupd8team/+archive/ubuntu/java).)
```

Comment:
What about "the list of published Oracle JDK packages" or similar, instead of "the list"?